src/components/Modal/VoteProposalModal.tsx

```diff
@@ -30,134 +51,139 @@ export const VoteProposalModal: FC<Props> = ({
-          <Paragraph className="text-sm text-gray-500 font-normal">
-            Proposed by:
-            <br />
-            <Span className="text-primary font-semibold">{shortAddress(proposal.proposer)}</Span>
-          </Paragraph>
-          <Paragraph className="text-sm text-gray-500 font-normal ml-4">
-            Created at:
-            <br />
-            <Span className="text-primary font-semibold">{proposal.Starts.format('YYYY-MM-DD')}</Span>
-          </Paragraph>
-          <Paragraph className="text-sm text-gray-500 ml-4 font-normal">
-            Proposal ID:
-            <br />
-            <Span className="text-primary font-semibold">{truncateMiddle(proposal.proposalId)}</Span>
-          </Paragraph>
+      <div className="p-8 flex flex-col min-h-[600px] bg-black">
+        <Typography className="text-5xl font-kk-topo leading-[48px] font-medium text-center w-full mb-12">
+          VOTING
+        </Typography>
+
+        <div className="flex justify-start space-x-16 items-start mb-12">
+          <div>
+            <Typography className="text-gray-400 text-base mb-1">Proposed by</Typography>
+            <Typography className="text-orange-500 text-base">{shortAddress(proposal.proposer)}</Typography>
+          </div>
+          <div>
+            <Typography className="text-gray-400 text-base mb-1">Proposal ID</Typography>
+            <Typography className="text-orange-500 text-base">
+              {truncateMiddle(proposal.proposalId)}
+            </Typography>
+          </div>
+          <div>
+            <Typography className="text-gray-400 text-base mb-1">Created on:</Typography>
+            <Typography className="text-orange-500 text-base">
+              {proposal.Starts.format('D MMM, YYYY')}
+            </Typography>
+          </div>
         </div>
-        <Label variant="semibold" className="mt-4">
-          Wallet
-        </Label>
-        <div className="p-[15px] bg-input-bg flex gap-2 items-center justify-between w-1/2 rounded-[6px]">
-          {/* @TODO insert provider image */}
-          <Image
-            src="/images/metamask.svg"
-            alt="metamask"
-            width={0}
-            height={0}
-            style={{ width: '22px', height: 'auto' }}
-          />
-          <Typography tagVariant="span" className="flex-1">
-            {shortAddress(address, 10)}
-          </Typography>
-          <button title="Copy address to clipboard" onClick={() => navigator.clipboard.writeText(address)}>
-            <FaCopy />
-          </button>
+
+        <div className="mb-12">
+          <Typography className="text-xl mb-1 font-medium">Wallet</Typography>
+          <div className="flex w-1/3 items-center gap-2 p-3 bg-[#1c1c1c] rounded">
+            <Typography className="flex-1 font-mono">{shortAddress(address)}</Typography>
+            <button onClick={handleCopy} className="text-[#2D2D2D]">
+              <FaCopy size={16} className="text-white" />
```

Comment:
Since you're editting this file, let's replace this `FaCopy` with the icon directly from the design. We're gradually removing the usage of `react-icons` library because it's causing very long compilation time in local environment.

Reply:
Good point. Done